### PR TITLE
Typo

### PR DESCRIPTION
**Pull request by oten**

### CODE REVIEW

**Comment by yashk2810** [comment]
You can also create a pull request for changing the Batchnorm layer's axis parameter to -1. As the code is using tensorflow as the backend, the axis parameter should be set to -1, which I haven't corrected yet.
